packages/story-editor/src/components/panels/document/taxonomies/HierarchicalTermSelector.js

```diff
@@ -131,14 +207,24 @@ function HierarchicalTermSelector({ taxonomy }) {
             id={dropdownId}
             ariaLabel={taxonomy.labels.parent_item}
             placeholder={taxonomy.labels.parent_item}
-            options={[]}
+            options={categories}
+            selectedValue={selectedParent}
+            onMenuItemClick={handleParentSelect}
           />
-          <AddNewCategoryButton onClick={handleAddCategory}>
-            {__('Add', 'web-stories')}
-          </AddNewCategoryButton>
+          <ButtonContainer>
+            <AddNewCategoryButton
+              disabled={!newCategoryName.length}
+              onClick={handleAddCategory}
+            >
+              {__('Add Category', 'web-stories')}
+            </AddNewCategoryButton>
+            <AddNewCategoryButton onClick={handleToggleNewCategory}>
+              {__('Cancel', 'web-stories')}
+            </AddNewCategoryButton>
+          </ButtonContainer>
         </AddNewCategoryArea>
       ) : (
-        <LinkButton onClick={handleShowAddNewCategoryClick}>
+        <LinkButton onClick={handleToggleNewCategory}>
           {taxonomy.labels.add_new_item}
         </LinkButton>
```

Comment:
This button shows the new category UI. Wondering if there's any extra accessibility that should be added to it to show users that it opens/displays new content

Reply:
Might need to use `aria-expanded` on the popup and `aria-controls` on the button?